google/cloud/storage/client.h

```diff
@@ -47,7 +47,9 @@ inline namespace STORAGE_CLIENT_NS {
  * copy-construction are also relatively low-cost operations, they should be
  * comparable to copying a few shared pointers. The first request (or any
  * request that requires a new connection) incurs the cost of creating the
- * connection and authenticating with the service.
+ * connection and authenticating with the service. Note that access tokens need
+ * to be refreshed before they expire, , the library
```

Comment:
Seems to be an extra space and comma here.

Reply:
Fixed.

Reply:
Sorry, I missed @frankyn comment.  Does this work?